index.bs

```diff
@@ -1142,10 +1149,16 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     [=biometric recognition=] (e.g., presenting a fingerprint) [[ISOBiometricVocabulary]]. The intent is to
     distinguish individual users.
 
+    Note that [=user verification=] does not give the [=[RP]=] a concrete identification of the user,
+    but when 2 or more ceremonies with [=user verification=] have been done with that [=credential=]
+    it expresses that it was the same user that performed all of them.
+    The same user might not always be the same natural person, however,
+    if multiple natural persons share access to the same [=authenticator=].
```

Comment:
perhaps add an example? 

"For example, by sharing the authentictor PIN."

Reply:
There's a Note: directly following with some examples. Is that enough? :slightly_smiling_face:

Reply:
doh!  :-P